resources/js/views/general/Tag.vue

```diff
@@ -87,7 +87,7 @@
                                     {{ $t('tags.to-tag') }}: {{ remaining }}
                                 </li>
                                 <li class="list-group-item">
-                                    {{ $t('tags.total-uploaded') }}: {{ user.photos_count }}
+                                    {{ $t('tags.total-uploaded') }}: {{ user.total_images }}
```

Comment:
We should be getting this data from Redis

Reply:
Yes, but the user's total_images is used elsewhere on the site too. We can get it from Redis when we migrate the other parts too. Besides, we can still use total_images as a user property, and change the implementation, or data source, on the user model on the backend, without changing anything here.